src/components/new-conversation-dialog/create-group/index.js

```diff
@@ -189,12 +190,31 @@ const CreateGroup = ({
             description: 'description for group members input',
           })}
           maxHeight={67}
-          tags={selectedMembers}
           onFocus={onTagsInputFocus}
           onRemove={onMemberRemove}
           isNecessityLabel
           className={classes.members}
-        />
+        >
+          {selectedMembers.map(tag => {
+            const { id, firstName, avatar, lastName, displayName } = tag
+            const name =
+              !firstName || !lastName ? displayName : `${firstName} ${lastName}`
```

Comment:
I know this has been in the code for some time but I don't think we actually need this - just use `displayName`. The backend already does the logic to come up with a good display name: https://github.com/ubergrape/chatgrape/blob/a4cc61de5660704d326f59a2a8291c8fa7f47a2e/accounts/models.py#L2607-L2621

Reply:
Ah, ok. Didn't know that, thanks!